discogs_client/exceptions.py

```diff
@@ -23,3 +23,10 @@ def __init__(self, message, code):
 
     def __str__(self):
         return self.msg
+
+
+class AuthorizationError(HTTPError):
+    """The server rejected the client's credentials."""
+    def __init__(self, message, code, response):
+        super(AuthorizationError, self).__init__(message, code)
+        self.msg = '%s Response: %s' % (self.msg, repr(response))
```

Comment:
Absolutely! I usually do that too and was just trying to conform. :smiley: I'll push that now.